examples/basicUserStore/user.store.ts

```diff
@@ -0,0 +1,21 @@
+import { Store } from "@leocode/rxstores";
+import { User } from "./user.model";
+
+export class UserStore extends Store<Partial<User>> {
+  constructor() {
+    const initialValue: Partial<User> = {};
+    super(initialValue);
```

Comment:
Then, since the first thing this store does on init is fetching some data from API, we can initialize the store with `Loadable.ofLoading()` state.